tests/ui/trait_duplication_in_bounds_unfixable.stderr

```diff
@@ -1,8 +1,8 @@
 error: this trait bound is already specified in the where clause
-  --> $DIR/trait_duplication_in_bounds_unfixable.rs:6:23
+  --> $DIR/trait_duplication_in_bounds_unfixable.rs:6:15
    |
 LL | fn bad_foo<T: Clone + Default, Z: Copy>(arg0: T, arg1: Z)
-   |                       ^^^^^^^
+   |               ^^^^^
```

Comment:
This and the next warning were both emitted with the order depending on a hash map. I guess there was a change recently that caused the order to change.